webapp/components/new_channel_modal.jsx

```diff
@@ -90,6 +90,7 @@ class NewChannelModal extends React.Component {
     handleChange() {
         const newData = {
             displayName: ReactDOM.findDOMNode(this.refs.display_name).value,
+            header: ReactDOM.findDOMNode(this.refs.channel_header).value,
```

Comment:
@DavidLu1997 by getting the reference with `this.refs.channel_header` you don't need ReactDOM, so feel free to remove it.

Also feel free to do the same thing with the other ones too.